ghedesigner/manager.py

```diff
@@ -874,7 +875,15 @@ def run_manager_from_cli_worker(input_file_path: Path, output_directory: Path) -
     show_default=False,
     help="Validate input and exit."
 )
-def run_manager_from_cli(input_path, output_directory, validate):
+@click.option(
+    "-f",
+    "--to-idf",
+    default=False,
+    is_flag=True,
+    show_default=False,
+    help="Convert output to EnergyPlus IDF objects."
+)
```

Comment:
New CLI option to convert outputs to IDF objects. I could use a review of whether this option name is OK or if there's a clearer way.

Reply:
I don't think it's bad or anything, but is it worth something like `--format=IDF`, which could also be `--format=EpJSON` or something else?

Reply:
Ah, I like that. I'll update it. I have some separate work where I'm attempting to convert to GLHEPro, so this would be useful.